### ISSUE TITLE
GPL-835 BUG - As sample management and reefer managers we need fix the issues with scanning 2D barcodes

### ISSUE THREAD

**Issue by stevieing** [sanger/labwhere#301]
**Describe the bug**
When users in the reefers try to scan 2D barcodes they will sometimes not scan although sample management say there is no issue.

**RT Ticket Number**
If applicable

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to page '...'
2. Click on button '...'
3. See error

**Expected behaviour**
What was expected to happen.

**Screenshots**

**Desktop (please complete the following information):**
* Windows or Mac?
* Browser Chrome, Firefox, Safari or other?
* Browser version (use 'About' to look up)?

**Additional context**

**Comment by stevieing** [comment]
Example barcodes:
w-unused-lighthouse-plates-94-19331
lw-lighthouse-cherrypicks-done-299-19154
lw-unused-lighthouse-plates-87-19146
lw-unused-lighthouse-plates-103-19340
lw-unused-lighthouse-plates-104-19341
lw-lighthouse-cherrypicks-done-259-18245

**Comment by stevieing** [comment]
I had a meeting with sample management on 20/10 and we tried to increase the size of the barcode. Although it worked printing multiples caused a problem and the barcode gets misaligned and won't scan.

We need to find a way to increase the size of the barcode and centre it.

**Comment by stevieing** [comment]
Once we have found a solution that works it would be a good idea to have a new option for printing barcodes 2D large so that they can run them in parallel to ensure confidence.

**Comment by stevieing** [comment]
This is now being packaged up as part of the Squix work.

**Comment by stevieing** [comment]
Closed as no longer needed.
